go/cmd/fixturegen/main.go

```
@@ -0,0 +1,88 @@
+	flag.Parse()
+
+	files, types, err := loadDescriptors(*protoDir)
+	check(err)
+	desc, err := files.FindDescriptorByName(protoreflect.FullName(*message))
+	check(err)
+	messageDesc, ok := desc.(protoreflect.MessageDescriptor)
+	if !ok {
+		check(fmt.Errorf("%s is not a message", *message))
+	}
+
+	check(os.MkdirAll(*out, 0o755))
```

golangci-lint (2.11.4)

[medium] G301: Expect directory permissions to be 0750 or less

(gosec)